src/openrct2/world/Map.cpp

```diff
@@ -63,8 +63,18 @@ using namespace OpenRCT2;
 /**
  * Replaces 0x00993CCC, 0x00993CCE
  */
-const CoordsXY CoordsDirectionDelta[] = { { -32, 0 },   { 0, +32 },   { +32, 0 },   { 0, -32 },
-                                          { -32, +32 }, { +32, +32 }, { +32, -32 }, { -32, -32 } };
+// clang-format off
+const std::array<CoordsXY, 8> CoordsDirectionDelta = {
+    CoordsXY{ -COORDS_XY_STEP, 0 },
+    CoordsXY{ 0, +COORDS_XY_STEP },
+    CoordsXY{ +COORDS_XY_STEP, 0 },
+    CoordsXY{ 0, -COORDS_XY_STEP },
```

Comment:
Might be more readable if this aligned up.

Reply:
What do you mean?

Reply:
if the comma was in the same location on each line it would be more readable

Reply:
If it was arranged like a table, with the x coordinates right beneath one another and the y coordinates too.

Reply:
Done